05-setup-loki-stack/README.md

```diff
@@ -70,39 +70,41 @@ Next, add the `Grafana` Helm repository and list the available charts:
 ```shell
 helm repo add grafana https://grafana.github.io/helm-charts
 
+helm repo update grafana
+
 helm search repo grafana
 ```
 
 The output looks similar to the following:
 
 ```text
-NAME                                            CHART VERSION   APP VERSION     DESCRIPTION                                       
-grafana/grafana                                 6.16.2          8.1.2           The leading tool for querying and visualizing t...
-grafana/enterprise-metrics                      1.5.0           v1.5.0          Grafana Enterprise Metrics                        
+NAME                                            CHART VERSION   APP VERSION     DESCRIPTION
+grafana/grafana                                 6.20.5          8.3.4           The leading tool for querying and visualizing t...
+grafana/enterprise-metrics                      1.7.2           v1.6.1         Grafana Enterprise Metrics
 grafana/fluent-bit                              2.3.0           v2.1.0          Uses fluent-bit Loki go plugin for gathering lo...
-grafana/loki                                    2.4.1           v2.1.0          Loki: like Prometheus, but for logs.
+grafana/loki                                    2.5.1           v2.1.0          Loki: like Prometheus, but for logs.
 ...
 ```
 
 **Notes:**
 
 - The chart of interest is `grafana/loki`, which will install standalone `Loki` on the cluster. Please visit the [loki-stack](https://github.com/grafana/helm-charts/tree/main/charts/loki-stack) page for more details about this chart.
```

Comment:
Should be `grafana/loki-stack` instead of `grafana/loki`. @vladciobancai seems that I made a typo here long time ago. Please make the necessary corrections. Thanks.

Reply:
Applied the recommendations